
nav_msgs/OccupancyGrid grid

# Array of obstacle pixel arrays
roadmaps_practice/ObPixels[] obs
